[2025-05-28T10:50:01.915+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_data_etl.transform_weather_task scheduled__2025-05-28T10:45:00+00:00 [queued]>
[2025-05-28T10:50:01.926+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_data_etl.transform_weather_task scheduled__2025-05-28T10:45:00+00:00 [queued]>
[2025-05-28T10:50:01.926+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-05-28T10:50:01.940+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): transform_weather_task> on 2025-05-28 10:45:00+00:00
[2025-05-28T10:50:01.946+0000] {standard_task_runner.py:60} INFO - Started process 1492 to run task
[2025-05-28T10:50:01.949+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'weather_data_etl', 'transform_weather_task', 'scheduled__2025-05-28T10:45:00+00:00', '--job-id', '53', '--raw', '--subdir', 'DAGS_FOLDER/weather.py', '--cfg-path', '/tmp/tmp4se6m88s']
[2025-05-28T10:50:01.951+0000] {standard_task_runner.py:88} INFO - Job 53: Subtask transform_weather_task
[2025-05-28T10:50:01.996+0000] {task_command.py:423} INFO - Running <TaskInstance: weather_data_etl.transform_weather_task scheduled__2025-05-28T10:45:00+00:00 [running]> on host f1bd559e8a8b
[2025-05-28T10:50:02.061+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_data_etl' AIRFLOW_CTX_TASK_ID='transform_weather_task' AIRFLOW_CTX_EXECUTION_DATE='2025-05-28T10:45:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-05-28T10:45:00+00:00'
[2025-05-28T10:50:02.223+0000] {python.py:201} INFO - Done. Returned value was: Data cuaca berhasil ditransformasi dengan 5 baris data
[2025-05-28T10:50:02.239+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=weather_data_etl, task_id=transform_weather_task, execution_date=20250528T104500, start_date=20250528T105001, end_date=20250528T105002
[2025-05-28T10:50:02.284+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-05-28T10:50:02.317+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
